WordPress/Classes/ViewRelated/Reader/ReaderMenuItemCreator.swift

```diff
@@ -0,0 +1,5 @@
+/// Interface abstracting the entities that create menu items for the different topics presented in Reader
+protocol ReaderMenuItemCreator {
```

Comment:
Because each class is so short, how would you feel about putting all of the `ReaderMenuItemCreator` implementations into a single swift file? (this one, I guess)

Reply:
I lean towards applying the single responsibility principle to files as well: one abstraction per file. I guess I tend to see short, granular files as better than longer files, slightly less cohesive.  I do not have very strong feelings about it though, so if you'd really prefer to move everything into a single file, I'm happy to do it 😃